high-level-caiman/src/lower/lower_schedule.rs

```diff
@@ -433,30 +420,30 @@ fn lower_select(guard_name: &str, tags: &TripleTag, temp_id: usize, f: &Funclet<
             },
         })),
         Some(asm::Command::TailEdge(asm::TailEdge::ScheduleSelect {
-            value_operation: Some(
+            operations: Some(vec![
```

Comment:
Just curious, but does order matter, or is the assembly using the meta-variables to determine which tag is for which spec.

Reply:
The assembly uses the meta-variables to determine which tag is for which spec, yep!